packages/orchestration/src/examples/shared/chain-hub-cf.js

```diff
@@ -0,0 +1,53 @@
+import { M } from '@endo/patterns';
+import { heapVowE } from '@agoric/vow/vat.js';
+import { CosmosChainInfoShape } from '../../typeGuards.js';
+
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {CosmosChainInfo, IBCConnectionInfo} from '@agoric/orchestration';
+ * @import {ChainHub} from '../../exos/chain-hub.js';
+ */
+
+/**
+ * Allows developers to add new chain configurations to a local chainHub, in the
```

Comment:
the chainHub already allows this. isn't the point of this is to create an endowment for async-flow?